app/manager-frontend/src/components/LowNumberSuppressionParameters.jsx

```diff
@@ -0,0 +1,51 @@
+import {
+  Button,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  VStack,
+  Alert,
+  AlertIcon,
+} from "@chakra-ui/react";
+
+import React, { useState, useEffect } from "react";
+import { Form, Formik, useFormikContext } from "formik";
+import { FaArrowRight } from "react-icons/fa";
+import { FormikInput } from "./forms/FormikInput";
+import { FormikSelect } from "./forms/FormikSelect";
+
+function LowNumberSuppressionParameters({ type }) {
+  const formikProps = useFormikContext();
+  useEffect(() => {
+    formikProps.setFieldValue("Parameters", { Threshold: null });
+  }, [type]);
+  // Todo: get the value of the inputs to update when the formik values update
+  switch (type) {
+    case "Type1":
+      return (
+        <Form noValidate>
```

Comment:
⛏️ Does the input need to be in a `Form` tag here, seeing as it's going to be inside a `Form` tag in the outer scope (`ConfigureResultsModifierModal`)?